### PR TITLE
vscode/settings.json markdown added

### PR DESCRIPTION
**Pull request by aadityaforwork**
Issue number 6354 resolved feedback required if any further changes are to be done.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,14 +36,17 @@
...
   "packageManager": "pnpm@9.1.1",
   "pnpm": {
     "overrides": {
       "@pmmmwh/react-refresh-webpack-plugin": "0.5.11",
       "react-refresh": "0.14.0"
     }
+  },
+  "dependencies": {
+    "parcel": "^2.12.0"
   }
```
Comment (davisagli) [inline package.json:51]: Please don't make changes to package versions that are unrelated to the main purpose of the pull request.

File: packages/volto/locales/ca/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -17,6 +17,11 @@ msgstr ""
 "Domain: plone\n"
 "X-Is-Fallback-For: ca-ad ca-es ca-fr ca-it\n"
 
+#. Default: "End Date"
+#: components/manage/Contents/Contents
+msgid " N"
+msgstr ""
```
Comment (davisagli) [inline packages/volto/locales/ca/LC_MESSAGES/volto.po:23]: Please don't make changes to the locales that are unrelated to the main purpose of the pull request.

**Review by davisagli — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (aadityaforwork) [reply package.json:51]: alright sorry will keep in mind!

File: packages/volto/locales/ca/LC_MESSAGES/volto.po (hunk 2 above)
Comment (aadityaforwork) [reply packages/volto/locales/ca/LC_MESSAGES/volto.po:23]: alright sorry will keep in mind!

**Comment by stevepiercy** [comment]
@aadityaforwork please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

**Comment by stevepiercy** [comment]
@davisagli I noticed that Plone packages with `plone/meta` have this thingie called [`.editorconfig`](https://github.com/plone/meta/blob/main/config/README.md#editorconfig). Should this setting go in that file or `.meta.toml`?

I was about to port this to plone.api and plone.restapi, and this gave me pause.

**Comment by davisagli** [comment]
@stevepiercy I don't see an `.editorconfig` setting available for requesting that autoformatting be entirely disabled.

**Review by davisagli — CHANGES_REQUESTED** [review]
@aadityaforwork Your PR still includes files that should not be here. I would expect changes only to vscodesettings.js, plus the file in packages/volto/news.

**Comment by aadityaforwork** [comment]
> @aadityaforwork Your PR still includes files that should not be here. I would expect changes only to vscodesettings.js, plus the file in packages/volto/news.

Hello @davisagli I have addressed the changes you asked if anything further is to be done please let me know!

File: packages/volto/news/6354.documentation
Comment (stevepiercy) [inline packages/volto/news/6354.documentation:1]: This file needs to be moved into `packages/scripts/news/`, per the failing CI check: https://github.com/plone/volto/actions/runs/11138321174/job/30953127809?pr=6357#step:12:13

I'm OK with the extension `.documentation` since it is "for documentation" per https://6.docs.plone.org/contributing/index.html#contributing-change-log-label, although you could argue `.internal`, too.

File: packages/volto/news/6354.documentation [hunk 3]
```diff
@@ -0,0 +1 @@
+Added the new configuration for Markdown in the vscodeSettingsJSON object @aadityaforwork
```
Comment (stevepiercy) [inline packages/volto/news/6354.documentation:1]: Change log notes should describe what the change actually does, and be from the perspective of the developer or user who reads it. Implementation details cloud the purpose.

```suggestion
Added the configuration for VSCode not to reformat Markdown and MyST files. @aadityaforwork
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
See review comments. Otherwise LGTM.

File: packages/volto/news/6354.documentation
Comment (aadityaforwork) [reply packages/volto/news/6354.documentation:1]: Final updates are done which you have asked! @stevepiercy

File: pnpm-lock.yaml
Comment (davisagli) [inline pnpm-lock.yaml:1]: There should be no changes in pnpm-lock.yaml since there are no changes to dependencies in package.json

**Review by davisagli — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (aadityaforwork) [reply pnpm-lock.yaml:1]: The line added and removed are the same although I discarded all the changes it is still showing an unnecessary file change!

Comment (davisagli) [reply pnpm-lock.yaml:1]: @aadityaforwork It's showing that you removed a newline character at the end of the file.

Should be ok, I'll merge once the current CI builds finish.

Comment (aadityaforwork) [reply pnpm-lock.yaml:1]: okay I will keep in mind next time thank you!

**Review by davisagli — APPROVED** [review]
